modules/administration-guide/partials/proc_defining-the-list-of-images-to-pull.adoc

```diff
@@ -9,14 +17,20 @@
 
 .Procedure
 
-. Get the list of relevant container images.
+. Gather a list of relevant container images for the {platforms-name} platform:
 +
 include::example$snip_{project-context}-getting-the-list-of-relevant-images.adoc[]
 
 
-. Exclude from the list the container images mounting volumes in Dockerfile.
-
-
+. Determine images from the list for pre-pulling. 
++
+For faster workspace startup times, consider pre-pulling the workspace-related images:
++
+* che-theia
```

Comment:
for CRW, image names are different.

See https://quay.io/organization/crw
or https://catalog.redhat.com/software/containers/search?q=codeready-workspaces-

* theia-rhel8
* theia-endpoint-rhel8
* pluginbroker-artifacts-rhel8
* pluginbroker-metadata-rhel8
* stacks-*-rhel8
* plugin-*-rhel8